### ISSUE TITLE
Setting body: "empty_body" is not working with 1.10.0 (and maybe even with 1.9.x)

### ISSUE THREAD

**Issue by tanvn** [palantir/bulldozer#207]
I created this issued before.
https://github.com/palantir/bulldozer/issues/192

And today I tried to update from 1.8.1 to 1.10.0 because of the deprecation Github API [here](https://developer.github.com/changes/2020-04-15-replacing-create-installation-access-token-endpoint/)

Here is my setting yaml file (unchanged from https://github.com/palantir/bulldozer/issues/192) 
```
version: 1

merge:
  whitelist:
    labels: ["merge-me"]
  blacklist:
    labels: ["do-not-merge","wip"]
  method: squash
  options:
    squash:
      title: "pull_request_title"
      body: "empty_body"
  delete_after_merge: true

update:
  blacklist:
    labels: ["do-not-update", "wip"]
  whitelist:
    labels: ["update-me"]
```

But the setting body: "empty_body" is ignored and 
as far as I understand, this [commit](https://github.com/palantir/bulldozer/pull/193/commits/0623c57ab2a210d7bd1d2d6dec51db3fda992afb
) (which is supposed to fix this issue) is still not be merged into the latest version.
https://github.com/palantir/bulldozer/compare/v1.10.0...develop

Please take a look. 
Because `the legacy "Create an installation access token" endpoint will be disabled at the earliest on October 1 2020.` so we would be very happy if you guys could fix this issue as soon as possible.

Thank in advance.

**Comment by bluekeyes** [comment]
Sorry about the delay, I just released 1.10.1 which includes this fix. In my testing, it resolves the issue, but please let us know if you still see any problems after upgrading.

**Comment by tanvn** [comment]
Thank you for your quick response !!!
I just tried updating to 1.10.1 and the message body when a PR is merged is now empty. 🎉 
Please keep up the great work !
